### PR TITLE
Updates for RMB 33.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: src/test/java/org/folio/rest/api/ItemStorageTest.java [hunk 1]
```diff
@@ -1991,7 +1991,7 @@ public void cannotCreateItemWithWrongStatus() throws Exception {
     Response postResponse = createCompleted.get(5, TimeUnit.SECONDS);
     assertThat(postResponse.getStatusCode(), is(400));
     assertThat(postResponse.getBody(),
-      matchesPattern("(?s)Json content error Cannot construct instance of `.+`, problem: Wrong status name.+")
+      matchesPattern("(?s)Cannot construct instance of `.+`, problem: Wrong status name.+")
```
Comment (julianladisch) [inline src/test/java/org/folio/rest/api/ItemStorageTest.java:1994]:
```suggestion
      containsString("problem: Wrong status name")
```

Comment (adamdickmeiss) [reply src/test/java/org/folio/rest/api/ItemStorageTest.java:1994]: Thx

**Comment by adamdickmeiss** [comment]
Closing as the PR has continued in #624

**Comment by julianladisch** [comment]
This PR is continued in #629

**Pull request closed without merging**